java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -23,8 +23,14 @@ public interface GenericBaseCommands {
      * Removes the specified <code>keys</code> from the database. A key is ignored if it does not
      * exist.
      *
-     * @apiNote When in cluster mode, the command may route to multiple nodes when <code>keys</code>
-     *     map to different hash slots.
+     * @apiNote In cluster mode, if keys in <code>keys</code> map to different hash slots,
+     * the command will be split across these slots and executed separately for each.
+     * This means the command is atomic only at the slot level. If one or more slot-specific
+     * requests fail, the entire call will return the first encountered error, even
+     * though some requests may have succeeded while others did not.
```

Comment:
If you want to make a line break:
```suggestion
     * though some requests may have succeeded while others did not.<br>
```